### ISSUE TITLE
Spam in logs

### ISSUE THREAD

**Issue by dannkunt** [v1a0/sqllex#12]
I think it shouldn't be shown if default arguments is set

```
SIZE CROP! Expecting 2 arguments but 1 were given!
Expecting: ['id', 'money']
Given: (340935787,)
```

**Comment by v1a0** [comment]
### Hey dannkunt, thanks for your report!

Yeah, now i see it might be annoying and I'm planning to leave this feature only for debug mode in the next updates.

As temporary fix recommend you to format your code like this:

```python
# From this
db.insert("your_table", 340935787) 

# To this
db.insert("your_table", id=340935787) 
```

I hope it'll help!

**Comment by v1a0** [comment]
# Fixed since Sqllex version 0.1.8.11

```
pip install sqllex -U
```
